### ISSUE TITLE
[med] JAWS+Chrome: Privacy Policy: Links are not descriptive enough to distinguish

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5505]
## Action Performed:
1. Using JAWS+Chrome, open https://staging.new.expensify.com/
2. Tab to focus the "privacy policy" link and press Enter key.
3. Tab to navigate through the links available on the page.
4. Check if the purpose of each link is clear.

## Expected Result:
The links text should be descriptive and unique, so that users can easily understand their purpose and distinguish between them.

## Actual Result:
There are several "click here" links on the page.

HTML:
`<a href="https://www.networkadvertising.org/choices"><span
style="text-decoration:underline">click here</span></a>
...
<a href="https://youradchoices.com/control"><span
style="text-decoration:underline">click here</span></a>`

## Workaround:
No workaround

## Area issue was found in:
Privacy policy page

## Failed WCAG checkpoints
2.4.4

## User impact:
The screen reader users will not be able to understand the links purpose from the link text alone or to distinguish between the links.

## Suggested resolution:
The best practice is to update the link text to make it unique and descriptive for each link. See: https://www.w3.org/WAI/WCAG21/Techniques/general/G91
If the above is not possible, use aria-label attribute to provide the context for each link. Refer to: https://www.w3.org/WAI/GL/wiki/Using_aria-label_for_link_purpose

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**

https://user-images.githubusercontent.com/88733897/134722462-c868ce5a-0236-4260-9ad9-2ba71c8f90d5.mp4

![Bug5182358_Link_purpose](https://user-images.githubusercontent.com/88733897/134722471-a80c786c-2a0e-42ca-b7db-47e4de1df19b.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by shawnborton** [comment]
Same comment: So the terms of service page is hosted on UseDot and it's technically not part of the NewDot app. @stitesExpensify @kadiealexander do you still think it's worth addressing these issues, or should we punt and focus on the app itself?
